network/src/config.rs

```diff
@@ -23,7 +23,7 @@ pub struct NetworkConfig {
     pub connect_outbound_interval_secs: u64,
     pub listen_addresses: Vec<Multiaddr>,
     pub public_addresses: Vec<Multiaddr>,
-    pub bootnodes: Vec<Multiaddr>,
+    pub bootnodes: Vec<String>,
```

Comment:
Multiaddr supports dns format, is it possible to keep Multiaddr here? User can mix different bootnodes in config:
```
bootnodes = [
"/ip4/47.111.175.189/tcp/8111/p2p/QmQqYVVigwiogARK4UgB5scgEd9tUfhSiZkLK7bM82cVCu",
"/dns/crystal-nova.rylai.nervos.org/"
]
```

Reply:
I'll separate them

Reply:
`/dns` is not an official Protocol, and I don't what choose between `/dns4` and `/dns6`.